src/Data/Data_Scoring.res

```diff
@@ -51,20 +51,20 @@ module Score = {
 
   let fromResultWhite = (x: Data_Match.Result.t) =>
     switch x {
-    | WhiteWon => One
-    | BlackWon => Zero
     | Draw => Half
     /* This loses data, so is a one-way trip. Use with prudence! */
-    | NotSet => Zero
+    | (WhiteWon | BlackAborted) => One
+    | (BlackWon | WhiteAborted) => Zero
+    | (NotSet | Aborted) => Zero
```

Comment:
This feels concerning

Reply:
If you mean the comment, you can actually just remove that. There was a point where I was converting the score type into match results, and that comment was a note to myself to be careful with the "NotSet" variant. I don't think that happens anywhere now, and doing it wouldn't make sense anyway.

Reply:
Yep, I was talking about the comment.
Removed.